CHANGELOG.rst

```diff
@@ -7,7 +7,11 @@ Rasa Change Log
 All notable changes to this project will be documented in this file.
 This project adheres to `Semantic Versioning`_ starting with version 1.0.
 
-
+[Unreleased 1.3.4] - `1.3.x`_```
```

Comment:
```suggestion
[Unreleased 1.3.4] - `1.3.x`_
```

Reply:
please make sure that the underlining has the correct length. Otherwise travis will fail 🙄